sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -345,6 +418,115 @@ internal static IReadOnlyList<EntityRecognitionTasksItem> ConvertToEntityRecogni
             return collection;
         }
 
+        private static Regex _targetRegex = new Regex("#/tasks/(\\w+)/(\\d+)", RegexOptions.Compiled, TimeSpan.FromSeconds(2));
```

Comment:
can you use the specific expected strings here instead of `w+`?

Reply:
also, i think it should locally defined in the `parseActionErrorTarget`. There is no need to have it as a class member.

Reply:
updated